### PR TITLE
add subgroups UI dropdown

### PR DESCRIPTION
**Pull request by spasovski**
No event bindings yet.

### CODE REVIEW

**Comment by openjck** [comment]
Ready for review and merge?

File: viewer/static/js/app/components/views/dataset-control.js [hunk 1]
```diff
@@ -33,3 +43,16 @@ export default function(props) {
...
+  <select className="dataset-subgroup-selection" value="all">
+    <option value="all">all</option>
+    {props.datasets.subgroups.map(subgroup => {
+      return (
+        <option key={subgroup} value={subgroup}>{subgroup}</option>
+      );
+    })}
+  </select>
+</div>
+*/
```
Comment (openjck) [inline viewer/static/js/app/components/views/dataset-control.js]: Was this left in accidentally?

**Comment by openjck** [comment]
Code looks great. Very clean and straightforward. Testing behavior now...

File: viewer/static/js/app/components/views/dataset-control.js (hunk 1 above)
Comment (spasovski) [inline viewer/static/js/app/components/views/dataset-control.js]: Yep I am a fool at times (most times).

Comment (openjck) [inline viewer/static/js/app/components/views/dataset-control.js]: If you were you'd be president

**Comment by openjck** [comment]
Hm. I'm still not seeing the charts update after selecting a different OS. Also, the selected OS reverts to *all*.

**Comment by openjck** [comment]
Looks good. Just waiting for Circle.

**Pull request merged**